index.js

```diff
@@ -14,59 +15,47 @@ module.exports = function (src, dest, opts, cb) {
 		opts = {};
 	}
 
-	cb = cb || function () {};
+	cb = onetime(cb) || function () {};
```

Comment:
That's almost the same, it only differs if `cb` is `undefined`: my would be negligible faster.